change/@fluentui-react-carousel-preview-533b9b0e-1769-4408-b096-4984d68abfaa.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the FluentUIV0 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32154&commitId=48b071aa65a4379a74db96546ded1af7098db80e&env=prod&clientType=FLUENTUI)


  <details><summary><b>TextArea</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextArea.Text Area Example Resize.chromium.png|0| Removed|
|TextArea.Text Area Example Teams.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example Teams.chromium.png|0| Removed|
|TextArea.Text Area Example.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>